packages/edit-post/src/hooks/commands/use-common-commands.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __, isRTL } from '@wordpress/i18n';
+import {
+	code,
+	cog,
+	drawerLeft,
+	drawerRight,
+	blockDefault,
+} from '@wordpress/icons';
+import { useCommand } from '@wordpress/commands';
+import { store as preferencesStore } from '@wordpress/preferences';
+import { store as interfaceStore } from '@wordpress/interface';
+
+/**
+ * Internal dependencies
+ */
+import { store as editPostStore } from '../../store';
+
+export default function useCommonCommands() {
+	const { openGeneralSidebar, closeGeneralSidebar, switchEditorMode } =
+		useDispatch( editPostStore );
+	const { editorMode, activeSidebar } = useSelect(
+		( select ) => ( {
+			activeSidebar: select( interfaceStore ).getActiveComplementaryArea(
+				editPostStore.name
+			),
+			editorMode: select( editPostStore ).getEditorMode(),
+		} ),
+		[]
+	);
+	const { toggle } = useDispatch( preferencesStore );
+
+	useCommand( {
+		name: 'core/open-settings-sidebar',
```

Comment:
I noticed that we're inconsistent with the command names in edit post and site packages. Some times we have the names like here and others like: `core/edit-site/reset-global-styles`. I mean it makes sense to have the extra prefix for core commands, but probably we should keep it simple for commands that only load in each editor. WDYT?